### PR TITLE
Brings back the USCM blurb on first drop

### PR DESCRIPTION
**Pull request by morrowwolf**
# About the pull request

This PR brings back the USCM blurb on first drop.

# Explain why it's good for the game

Lost during shuttle rework.

# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog

:cl: Morrow
add: Brought back the USCM blurb on first drop
/:cl:

### CODE REVIEW

**Comment by ihatethisengine** [comment]
Re-add "Unidentified lifesigns detected onboard. Recommendation: lockdown of exterior access ports, including ducting and ventilation." message, was also lost during rework

**Comment by morrowwolf** [comment]
> Re-add "Unidentified lifesigns detected onboard. Recommendation: lockdown of exterior access ports, including ducting and ventilation." message, was also lost during rework

AHEM https://github.com/cmss13-devs/cmss13/pull/3558

**Comment by ihatethisengine** [comment]
Damn you're quick!

**Review by harryob — APPROVED** [review]

File: code/modules/shuttle/shuttles/dropship.dm [hunk 1]
```diff
@@ -200,6 +205,10 @@
 		xeno_announcement(SPAN_XENOANNOUNCE("The dropship has landed."), "everything")
 		xeno_announce = FALSE
 
+	if(SSticker?.mode && !(SSticker.mode.flags_round_type & MODE_DS_LANDED))
+		SSticker.mode.flags_round_type |= MODE_DS_LANDED
+		SSticker.mode.ds_first_landed(src)
```
Comment (fira) [inline code/modules/shuttle/shuttles/dropship.dm:210]: Won't that double up with the first_drop_complete above??

**Review by fira — APPROVED** [review]
It's a bit weird to have handling split between mode and SSobjectives but i guess that works

**Pull request merged**